src/assets/toolkit/scripts/lib/component/fact-slider.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+export class FactSlider {
+  constructor (element, {
+    facts = element.querySelectorAll('.FunFact'),
+    factNavNext = element.querySelector('.js-FunFact-next'),
+    factNavPrev = element.querySelector('.js-FunFact-prev'),
+    factNavCurrent = element.querySelector('.js-FunFact-current'),
+    factNavTotal = element.querySelector('.js-FunFact-total'),
+    className = 'is-visible'
+  } = {}) {
+
+    facts = Array.from(facts);
+    let numFacts = facts.length;
+    let counter = 0;
```

Comment:
Instead of defining this stuff here and then using `Object.assign` to attach it to `this`, you could just do...

```js
this.numFacts = facts.length;
this.counter = 0;
```